docs/deep-dives/specifications/fml/fml-cli.mdx

```diff
@@ -96,3 +108,76 @@ To help this, the `single-file` command is used to merge all imports and include
 ```
 % nimbus-fml single-file --channel release @mozilla-mobile/firefox-android/fenix/app/nimbus.fml.yaml single-file.fml.yaml
 ```
+
+## Generating a machine readable overview of a feature
+
+```sh
+nimbus-fml info [--channel CHANNEL] [--json] <MANIFEST> [--feature FEATURE]
+```
+
+This prints a simplified YAML or JSON representation for each feature. e.g.
+
+```json
+{
+  "file": "https://raw.githubusercontent.com/mozilla-mobile/firefox-android/main/fenix/app/nimbus.fml.yaml",
+  "features": {
+    "homescreen": {
+      "description": "The homescreen that the user goes to when they press home or new tab.",
+      "types": [
+        "Boolean",
+        "HomeScreenSection",
+        "Map<HomeScreenSection, Boolean>"
+      ],
+      "hashes": {
+        "schema": "7a15570b",
+        "defaults": "423ab0bd"
+      }
+    }
+  }
+}
+```
+
+The `hashes` object shows truncated SHA256 hashes for:
```

Comment:
Nit: add subheadings

```suggestion
### Hashes

The `hashes` object shows truncated SHA256 hashes for:
```

Reply:
I think adding these would make the heading previews nicer:
<img width="443" alt="image" src="https://github.com/mozilla/experimenter-docs/assets/43795363/71a29b2e-ed73-4474-8fec-0d872533a342">